### PR TITLE
pass into locale to usages of getExpression function

### PR DESCRIPTION
**Pull request by musangowope**
- Change key of constitutionData to `en` aligns with i18next country codes
- passed in local to getExpressions
- refactored getExpression, so if the language key is results to nothing, data by default accesses the english content. (something constitution.json show always have an English key)

### CODE REVIEW

**Comment by musangowope** [comment]
@longhotsummer could we chat through this PR when you have capacity?

File: src/pages/constitution/Constitution_Full.tsx [hunk 1]
```diff
@@ -52,8 +52,7 @@ class Constitution_Full extends React.Component<Props, MyState> {
     };
 
     // parse the constitution HTML once
-    // TODO: language
-    this.constitution = getExpression('eng');
+    this.constitution = getExpression(localStorage.getItem('locale') || 'en');
```
Comment (longhotsummer) [inline src/pages/constitution/Constitution_Full.tsx:55]: move into componentDidMount, do event handlers after cloning.

File: src/data/search.tsx [hunk 2]
```diff
@@ -142,8 +142,7 @@ export function findTopicsByProvisionId(id: string) {
   return result;
 }
 
-// TODO: language for search?
-const constitution = getExpression('eng');
+const constitution = getExpression(localStorage.getItem('locale') || 'en');
```
Comment (longhotsummer) [inline src/data/search.tsx:145]: move this into searchLunr.

1. searchLunr must be passed a language, which is the current UI language
2. searchLurnr must check if its search data is in that language; if nothing has been indexed, or if it's the wrong language, re-index
3. perform the search

File: src/pages/constitution/Constitution_Full.tsx (hunk 1 above)
Comment (musangowope) [reply src/pages/constitution/Constitution_Full.tsx]: done

File: src/data/search.tsx (hunk 2 above)
Comment (musangowope) [reply src/data/search.tsx]: Done. Please me know if I did this correctly, or if I have missed out on a step

**Review by longhotsummer — APPROVED** [review]
nice

**Pull request merged**